### PR TITLE
Learning Materials section:  tutorials, field guides, publications, and StanCon talks

### PR DESCRIPTION
**Pull request by mitzimorris**
This PR incorporates and extends branch `vianey-edits`.

* Tutorials section lists two tutorials, plus a bunch of YouTube videos and podcasts
* Section "field guides" renamed to "Domain-specific" - information on the various Stan field guides plus StanConnect info ~~(note: StanConnect sections need links to YouTube videos, where available, and programs, etc)~~   Assembled programs for all StanConnect events and links to recordings if available
* Section "articles" renamed to "publications"
* Section "events" renamed to "StanCon talks" - provides listings of all StanCons in reverse chronological order.  Assembled programs for all events, added links to recordings

### CODE REVIEW

File: learn-stan/tutorials.qmd [hunk 1]
```diff
@@ -4,6 +4,39 @@ page-layout: full
...
+## Videos
+
+The official Stan youtube channel is:  [https://www.youtube.com/channel/UCwgN5srGpBH4M-Zc2cAluOA](https://www.youtube.com/channel/UCwgN5srGpBH4M-Zc2cAluOA){target="_blank"}
+
+### Introductory Videos
+
+* [Stan Tutorials](https://www.youtube.com/playlist?list=PLCrWEzJgSUqwL85xIj1wubGdY15C5Gf7H){target="_blank"} &nbsp; Maggie Lieu.
+A series of 8 videos that will get you started using Stan in R and Python.
+
+### Contributed Videos
```
Comment (WardBrian) [inline learn-stan/tutorials.qmd:24]: How strong do we want to make this distinction?

File: learn-stan/tutorials.qmd [hunk 2]
```diff
@@ -4,6 +4,39 @@ page-layout: full
 toc-location: left
 ---
 
-# Tutorials
+## Tutorials
+
+* [Getting Started with Bayesian Statistics using Stan and Python](https://github.com/bob-carpenter/stan-getting-started){target="_blank"} &nbsp;Bob Carpenter
+
+* [StanCon2023/Stan_tutorial.ipynb](https://github.com/charlesm93/stanTutorial/blob/main/StanCon2023/Stan_tutorial.ipynb) &nbsp;Charles Margossian
```
Comment (WardBrian) [inline learn-stan/tutorials.qmd]:
```suggestion
* [StanCon2023/Stan_tutorial.ipynb](https://github.com/charlesm93/stanTutorial/blob/main/StanCon2023/Stan_tutorial.ipynb){target="_blank"} &nbsp;Charles Margossian
```

File: learn-stan/articles.qmd [hunk 3]
```diff
@@ -74,33 +75,39 @@ toc-location: left
 
 ## Bayesian Workflow
 
-* Workflow paper
+* Andrew Gelman, Aki Vehtari, Daniel Simpson, Charles C. Margossian, Bob Carpenter, Yuling Yao, Lauren Kennedy, Jonah Gabry, Paul-Christian Bürkner, Martin Modrák. 2020. [Bayesian Workflow](https://arxiv.org/pdf/2011.01808)
 
 * Aki papers - PSIS-LOO-CV                       
 
                        
 ## Books using Stan
```
Comment (WardBrian) [inline learn-stan/articles.qmd]: Should we add Active Statistics and Regression and Other Stories?

File: learn-stan/field-guides.qmd [hunk 4]
```diff
@@ -1,22 +1,73 @@
...
+    + Maintainer: [Bruno Nicenboim](https://www.tilburguniversity.edu/staff/b-nicenboim)
+
+
+
+## StanConnect
+
+StanConnect is a virtual miniseries which consists of 3-hour meetings/mini-symposia
+where each meeting is like "session" of an organized conference.
+This series was started in 2021 and reprised in 2022.
+Future StanConnect series will be announced on [Discourse](https://discourse.mc-stan.org/tag/stanconnect).
```
Comment (WardBrian) [inline learn-stan/field-guides.qmd:45]:
```suggestion
Future entries in the StanConnect series will be announced on [Discourse](https://discourse.mc-stan.org/tag/stanconnect).
```
?

File: learn-stan/field-guides.qmd [hunk 5]
```diff
@@ -1,22 +1,73 @@
...
+
+
+### Stan Through Space and Time (2022)
```
Comment (WardBrian) [inline learn-stan/field-guides.qmd:48]: We should link to the resources from these if we have them, e.g.

2022:
https://github.com/stan-dev/connect22-space-time 

2021:
https://www.youtube.com/watch?v=Ie0WGa6vAws
https://www.youtube.com/watch?v=yn7jIHzvqXU
https://www.youtube.com/watch?v=cq5r0sSTPIg
https://www.youtube.com/watch?v=rsY3dVAP9gQ

File: stan-website.Rproj
Comment (WardBrian) [inline stan-website.Rproj:1]: I don't think this file should be checked in?

**Review by WardBrian — APPROVED** [review]
A few small things but mostly I think these changes look really good

File: learn-stan/tutorials.qmd (hunk 1 above)
Comment (mitzimorris) [reply learn-stan/tutorials.qmd:24]: not really necessary.  the YouTube channel has these labels - we don't need to mention them.

File: learn-stan/tutorials.qmd (hunk 2 above)
Comment (mitzimorris) [reply learn-stan/tutorials.qmd]: fixed - added {target="_blank"} everywhere.

File: learn-stan/field-guides.qmd (hunk 4 above)
Comment (mitzimorris) [reply learn-stan/field-guides.qmd:45]: agree.

File: learn-stan/field-guides.qmd (hunk 5 above)
Comment (mitzimorris) [reply learn-stan/field-guides.qmd:48]: linked to all of them.

File: learn-stan/articles.qmd (hunk 3 above)
Comment (mitzimorris) [reply learn-stan/articles.qmd]: added.

**Pull request merged**